### PR TITLE
Polars: add `make_private_agg`

### PR DESCRIPTION
**Pull request by ankke**

### CODE REVIEW

**Comment by ankke** [comment]
Current dependencies on/for this PR:
* main
  * **PR #763** <a href="https://app.graphite.dev/github/pr/opendp/opendp/763" target="_blank">Graphite</a> 
    * **PR #765** <a href="https://app.graphite.dev/github/pr/opendp/opendp/765" target="_blank">Graphite</a> 
      * **PR #767** <a href="https://app.graphite.dev/github/pr/opendp/opendp/767" target="_blank">Graphite</a> 
        * **PR #769** <a href="https://app.graphite.dev/github/pr/opendp/opendp/769" target="_blank">Graphite</a> 
          * **PR #771** <a href="https://app.graphite.dev/github/pr/opendp/opendp/771" target="_blank">Graphite</a> 
            * **PR #774** <a href="https://app.graphite.dev/github/pr/opendp/opendp/774" target="_blank">Graphite</a> 
              * **PR #795** <a href="https://app.graphite.dev/github/pr/opendp/opendp/795" target="_blank">Graphite</a> 
                * **PR #819** <a href="https://app.graphite.dev/github/pr/opendp/opendp/819" target="_blank">Graphite</a> 
                  * **PR #829** <a href="https://app.graphite.dev/github/pr/opendp/opendp/829" target="_blank">Graphite</a> 
                    * **PR #854** <a href="https://app.graphite.dev/github/pr/opendp/opendp/854" target="_blank">Graphite</a> 
                    * **PR #797** <a href="https://app.graphite.dev/github/pr/opendp/opendp/797" target="_blank">Graphite</a> 
                      * **PR #824** <a href="https://app.graphite.dev/github/pr/opendp/opendp/824" target="_blank">Graphite</a> 
                        * **PR #847** <a href="https://app.graphite.dev/github/pr/opendp/opendp/847" target="_blank">Graphite</a> 
                          * **PR #841** <a href="https://app.graphite.dev/github/pr/opendp/opendp/841" target="_blank">Graphite</a> 
                            * **PR #856** <a href="https://app.graphite.dev/github/pr/opendp/opendp/856" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/opendp/opendp/856?utm_source=stack-comment).

**Comment by ankke** [comment]
Current dependencies on/for this PR:
* main
  * **PR #763** <a href="https://app.graphite.dev/github/pr/opendp/opendp/763" target="_blank">Graphite</a> 
    * **PR #765** <a href="https://app.graphite.dev/github/pr/opendp/opendp/765" target="_blank">Graphite</a> 
      * **PR #767** <a href="https://app.graphite.dev/github/pr/opendp/opendp/767" target="_blank">Graphite</a> 
        * **PR #769** <a href="https://app.graphite.dev/github/pr/opendp/opendp/769" target="_blank">Graphite</a> 
          * **PR #771** <a href="https://app.graphite.dev/github/pr/opendp/opendp/771" target="_blank">Graphite</a> 
            * **PR #774** <a href="https://app.graphite.dev/github/pr/opendp/opendp/774" target="_blank">Graphite</a> 
              * **PR #795** <a href="https://app.graphite.dev/github/pr/opendp/opendp/795" target="_blank">Graphite</a> 
                * **PR #819** <a href="https://app.graphite.dev/github/pr/opendp/opendp/819" target="_blank">Graphite</a> 
                  * **PR #829** <a href="https://app.graphite.dev/github/pr/opendp/opendp/829" target="_blank">Graphite</a> 
                    * **PR #854** <a href="https://app.graphite.dev/github/pr/opendp/opendp/854" target="_blank">Graphite</a> 
                    * **PR #797** <a href="https://app.graphite.dev/github/pr/opendp/opendp/797" target="_blank">Graphite</a> 
                      * **PR #824** <a href="https://app.graphite.dev/github/pr/opendp/opendp/824" target="_blank">Graphite</a> 
                        * **PR #847** <a href="https://app.graphite.dev/github/pr/opendp/opendp/847" target="_blank">Graphite</a> 
                          * **PR #841** <a href="https://app.graphite.dev/github/pr/opendp/opendp/841" target="_blank">Graphite</a> 
                            * **PR #856** <a href="https://app.graphite.dev/github/pr/opendp/opendp/856" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/opendp/opendp/856?utm_source=stack-comment).

File: rust/src/measurements/polars/make_agg_meas/mod.rs [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+pub fn make_agg_meas<MI, MO>(
+    input_domain: LazyGroupByDomain,
+    input_metric: MI,
+    measurement: Measurement<ExprDomain<LazyGroupByContext>, Vec<Expr>, MI, MO>,
+) -> Fallible<Measurement<LazyGroupByDomain, LazyFrame, MI, MO>>
+where
+    (LazyGroupByDomain, MI): MetricSpace,
+    MI: Metric + Send + Sync + 'static,
+    MO: Measure + Send + Sync + 'static,
+    <MI as Metric>::Distance: Zero,
```
Comment (Shoeboxam) [inline rust/src/measurements/polars/make_agg_meas/mod.rs:17]: I don't think this bound is necessary.

**Comment by Shoeboxam** [comment]
Closing in favor of #847 (the group by and agg PRs are now one).

**Pull request closed without merging**